packages/server/email/MailManagerDebug.ts

```diff
@@ -7,6 +8,11 @@ export default class MailManagerDebug extends MailManager {
     To: ${to}
     Subject: ${subject}
     Body: ${body}`)
+
+    const {html} = options
+    const filename = `/tmp/${to}-${subject.replaceAll(' ', '-')}.html`
+    fs.writeFileSync(filename, html)
```

Comment:
@Dschoordsch I'm getting `Error: ENAMETOOLONG: name too long` errors in dev. shall we truncate the `to` field to 50 chars or so? first letter of everyones name?

Reply:
#8391